modules/openapi-generator/src/main/resources/r/api.mustache

```diff
@@ -95,10 +95,15 @@
       
       if (httr::status_code(resp) >= 200 && httr::status_code(resp) <= 299) {
         {{#returnType}}
-        returnObject <- {{returnType}}$new()
-        result <- returnObject$fromJSON(httr::content(resp, "text", encoding = "UTF-8"))
-        Response$new(returnObject, resp)
-        {{/returnType}}
+          {{#isPrimitiveType}}
+                returnObject <- {{returnType}}$new()
+                result <- returnObject$fromJSON(httr::content(resp, "text", encoding = "UTF-8"))
+                Response$new(returnObject, resp)
+           {{/isPrimitiveType}}
+          {{^isPrimitiveType}}
```

Comment:
@jokedurnez thanks for the PR. Seems like this line is missing a space in terms of alignment?